media/webview.css

```diff
@@ -76,7 +75,7 @@ BDO[DIR="rtl"]  { direction: rtl; unicode-bidi: bidi-override }
 /* Elements that are block-level in HTML4 */
 ADDRESS, BLOCKQUOTE, BODY, DD, DIV, DL, DT, FIELDSET,
 FORM, FRAME, FRAMESET, H1, H2, H3, H4, H5, H6, IFRAME,
-NOSCRIPT, NOFRAMES, OBJECT, OL, P, UL, APPLET, CENTER,
```

Comment:
https://sonarcloud.io/project/issues?id=vscode-didact&issues=AXR4WpWr5CyWyJNY-AoC&open=AXR4WpWr5CyWyJNY-AoC

Reply:
Will deal with this in a separate PR after https://github.com/redhat-developer/vscode-didact/pull/291 gets merged